packages/apollo-server-lambda/src/lambdaApollo.ts

```diff
@@ -39,36 +43,74 @@ export function graphqlLambda(
         statusCode: 500,
       });
     }
-    runHttpQuery([event, context], {
-      method: event.httpMethod,
-      options: options,
-      query:
-        event.httpMethod === 'POST' && event.body
-          ? JSON.parse(event.body)
-          : event.queryStringParameters,
-      request: {
-        url: event.path,
+
+    const runQuery = (query:any) => {
```

Comment:
Would the type of this `query` be a `HttpQueryRequest`?: 

https://github.com/apollographql/apollo-server/blob/2728b9be5e1f84ea0abebcc97bb60082f82fd8ed/packages/apollo-server-core/src/runHttpQuery.ts#L24